src/main/scala/com/gu/subscriptions/cas/service/DataStore.scala

```diff
@@ -0,0 +1,56 @@
+package com.gu.subscriptions.cas.service
+
+import com.amazonaws.auth.{AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain}
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsyncClientBuilder
+import com.gu.scanamo.DynamoFormat
+import com.gu.subscriptions.cas.service.api.{Error, GetExpirationResponse, SetExpirationResponse, Success}
+import org.joda.time.{DateTime, DateTimeZone, ReadablePeriod}
+import com.gu.scanamo._
+import com.gu.scanamo.syntax._
+import com.gu.subscriptions.cas.config.Configuration
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class DataStore(implicit ec: ExecutionContext) extends api.DataStore {
+
+  case class AuthItem(appId: String, deviceId: String, expiryDate: DateTime, ttlTimestamp: Option[Long])
+
+  private lazy val credentialsProvider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("membership"),
+    new DefaultAWSCredentialsProviderChain()
+  )
+
+  val dynamoClient = AmazonDynamoDBAsyncClientBuilder.standard()
+    .withRegion(Regions.EU_WEST_1)
+    .withCredentials(credentialsProvider)
+    .build()
+
+  implicit val jodaStringFormat = DynamoFormat.coercedXmap[DateTime, String, IllegalArgumentException](
```

Comment:
Consider switching this to a LocalDate as times are not necessary for expiry validation.